backend/src/openbeheer/zaaktype/api/views.py

```diff
@@ -266,13 +277,41 @@ def expand_deelzaaktype(
 ) -> list[list[ZaakType | None]]:
     return [
         [
-            fetch_one(client, dz_url, ZaakType) if dz_url else None
+            fetch_one(client, dz_url, ZaakTypeWithUUID) if dz_url else None
             for dz_url in (zt.deelzaaktypen or [])
         ]
         for zt in zaaktypen
     ]
 
 
+# TODO: If we are going to expand the selectielijst fields of the expanded resources,
+# we will need to revisit this because we shouldn't construct the client everytime.
```

Comment:
Not now. But I think we do something smart with caching by prefixes of the urls.
As for this expand api: maybe the cached clients can live in an APIClient facade that will  we can pass around instead of the ztc_client.